src/engine.cpp

```diff
@@ -1112,9 +1112,9 @@ void EnginesMonthlyLoop()
 	}
 }
 
-static IntervalTimer<TimerGameCalendar> _engines_monthly({TimerGameCalendar::MONTH, TimerGameCalendar::Priority::ENGINE}, [](auto)
+static IntervalTimer<TimerGameCalendar> _calendar_engines_monthly({TimerGameCalendar::MONTH, TimerGameCalendar::Priority::ENGINE}, [](auto)
 {
-	EnginesMonthlyLoop();
+	EnginesCalendarMonthlyLoop();
```

Comment:
My OCD hits on the difference in where "Calendar" goes. `_calendar_engines_monthly` vs `EnginesCalendarMonthlyLoop`. Pick one please :) Not both.